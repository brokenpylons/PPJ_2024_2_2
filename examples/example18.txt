for (var i = 0, 3) {
    for (var j = 0, 3) {
        print(i + j);
    }
    
}
