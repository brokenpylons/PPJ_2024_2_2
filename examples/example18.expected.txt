lparen("(") real("2") pow("^") real("3") rparen(")") pow("^") real("4") 
